### ISSUE TITLE
maweb zero position not initialized

### ISSUE THREAD

**Issue by GralfR** [cbdevnet/midimonster#74]
Hi!

I've this config for mapping MIDI-CC to MA2:
```
[backend winmidi]

[winmidi bcf]
read = BCF2000
write = BCF2000

[backend maweb]
interval = 100

[maweb MA]
host = 192.168.0.85 80
user = BCF

[map]
bcf.ch0.cc{1..8} <> MA.page1.fader{1..8}
```
Starting MIDmonster moves the faders on the controller to the positions in onPC. the mapping is working except for the initialising. If I put my faders on the controller into a non-zero position first and then start midimonster, faders on the controller are not set to the position of executors in MA if the executor-fader is at 0%.
This could be confirmed, if I use a lua script and print the channel/input_value. The executors at zero are not beeing set at startup of Midimonster.
I'm using the 0.5 or 0.5.27 for windows.

**Comment by cbdevnet** [comment]
Hi @GralfR, thanks for this report!

You're right, the backend should initially push an update for any value, not just any value that is not `0`.
I'll push a fix soon.

**Comment by cbdevnet** [comment]
This should be fixed with 7f2f36ddaa9c8311404248c69cc686bac6264ceb
